lib/rexml/element.rb

```diff
@@ -1505,7 +1505,7 @@ def write(output=$stdout, indent=-1, transitive=false, ie_hack=false)
       Kernel.warn("#{self.class.name}.write is deprecated.  See REXML::Formatters", uplevel: 1)
       formatter = if indent > -1
           if transitive
-            require_relative "formatters/transitive"
+            require_relative "formatters/transitive"  
```

Comment:
This position is lazy `require`.
Please move this at the beginning of this file so that it is non-lazy `require`.

https://github.com/ruby/rexml/blob/b70388c2638d90ebd2ae471bd85239d8469b8e62/lib/rexml/element.rb#L1-L8